src/devices/CpuTemperature/CpuTemperature.cs

```diff
@@ -3,29 +3,64 @@
 // See the LICENSE file in the project root for more information.
 
 using System;
+using System.Collections.Generic;

```

Comment:
I think in this case because implementation is completely different I'd recommend to create platform specific files: CpuTemperature.Linux.cs, CpuTemperature.Windows.cs

Also you should not have public API differences between Windows/Linux, extra stuff can be available through Windows-only library though

Reply:
The Iot.Devices library has no platform dependency (and shouldn't have one). Therefore the platform test is done at runtime, so that two files don't really add value, I think. 

The API is exactly the same for all cases.